contrib/client_ssl_auth/filters/network/test/client_ssl_auth_test.cc

```diff
@@ -167,7 +167,7 @@ TEST_F(ClientSslAuthFilterTest, Ssl) {
               setResponseFlag(StreamInfo::CoreResponseFlag::UpstreamProtocolError));
   EXPECT_CALL(filter_callbacks_.connection_.stream_info_,
               setResponseCodeDetails("auth_digest_no_match"));
-  EXPECT_CALL(filter_callbacks_.connection_, close(Network::ConnectionCloseType::NoFlush));
+  EXPECT_CALL(filter_callbacks_.connection_, close(Network::ConnectionCloseType::NoFlush, _));
```

Comment:
Please validate that the reason is `"auth_digest_no_match"`

Reply:
Thanks. Added it